### ISSUE TITLE
About the backbone

### ISSUE THREAD

**Issue by sunnyHelen** [NorthSummer/LGKD#1]
Hi, I read your paper on Arxiv and the performance is very impressive. I wonder if the ResNet-18 you used was pretrained on other datasets before (ImageNet or other detection datasets). Looking forward to your reply~

**Comment by NorthSummer** [comment]
Hi, thanks for your attention; yes, the student backbone is pretrained on ImageNet. Besides, the student backbone is also trained on nuScenes for 23 epochs when CBGS is not adopted. With CBGS adopted, the student backbone is only pre-trained on ImageNet.
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Tue, Jan 10, 2023 11:11 AM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; 
Subject: &nbsp;[NorthSummer/LGKD] About the backbone (Issue #1)

&nbsp;



 
Hi, I read your paper on Arxiv and the performance is very impressive. I wonder if the ResNet-18 you used was pretrained on other datasets before (ImageNet or other detection datasets). Looking forward to your reply~

**Comment by NorthSummer** [comment]
Hi, thanks for your attention. Yes, the student backbone is pretrained on ImageNet. Besides, the student backbone is also trained on nuScenes for 23 epochs when CBGS strategy is not adopted. With CBGS strategy adopted, the student backbone is only pre-trained on ImageNet.

**Comment by sunnyHelen** [comment]
Got it. Thanks for your reply.

**Comment by sunnyHelen** [comment]
Hi, it's me again. Could you let me know the weights you took for Lbev and Ld? I didn't find them in your paper. Moreover, could you please give some detail on how to realize the soft label distillation loss Lsoft?
